lib/src/transaction_builder.dart

```diff
@@ -90,6 +92,25 @@ class TransactionBuilder {
     return _tx.addOutput(scriptPubKey, value);
   }
 
+  int addNullOutput(dynamic data) {
```

Comment:
Using dynamic is ill-advised but it is consistent with the library interface. Ideally outputs will later be provided as an `Output` object to a single `addOutput` method.

Reply:
Keeping the `dynamic` interface for now.